ras_party/support/session_decorator.py

```diff
@@ -49,12 +32,7 @@ def handle_query_only_session(f, args, kwargs):
         result = f(*args, session=session, **kwargs)
         return result
     except SQLAlchemyError:
-        logger.error("Something went wrong accessing database",
-                     pool_size=current_app.db.engine.pool.size(),
-                     connections_in_pool=current_app.db.engine.pool.checkedin(),
-                     connections_checked_out=current_app.db.engine.pool.checkedout(),
-                     current_overflow=current_app.db.engine.pool.overflow(),
-                     exc_info=True)
+        logger.error("Something went wrong accessing database", exc_info=True)
```

Comment:
Does this do the same as logger.exception?

Reply:
Yeah, but it puts the stack trace as a field inside the log line instead of putting it over multiple lines.  The latter looks good when developing locally but looks like crap when looking at the logs in a log aggregator